components/dictionary.txt

```diff
@@ -3,9 +3,19 @@ dataset
 dataset's
 DESeq
 DESeq2
+ensembl
 et.
+KEGG
+medulloblastoma
 PNG
+ribosomes
 `.Rmd`
 RStudio
+softwares
```

Comment:
I think this is error comes from the template:

https://github.com/AlexsLemonade/refinebio-examples/blob/7b90e967f96ed6846cb77c40bcdeb48bb3fc13ea/template/template_example.Rmd#L223

Where I would expect that to say `software`. I might be wrong.

Reply:
It has been in the template this whole time, but I for whatever reason thought `softwares` was an acceptable word.

Reply:
> The plural form of software is software. 

[ref](https://www.dictionary.com/e/is-software-a-mass-noun/)

Reply:
Yes, it should not be included in the dictionary and I would file an issue about remedying it throughout the project (similar to #243).

Reply:
#244